### PR TITLE
Added & modified the rest of the contents

### PR DESCRIPTION
**Pull request by psy2848048**

### CODE REVIEW

File: content/docs/howto/create_your_own_pair.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+weight: 22
+bookFlatSection: true
+---
+
+# Create Your Own Pair
+
+PLEASE CHECK [HERE](#important---only-for-terra-20) for additional action on Terra 2.0
```
Comment (jhlee-young) [inline content/docs/howto/create_your_own_pair.md]: the link should be fixed

**Review by caramis — APPROVED** [review]
LGTM

File: content/docs/howto/token.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+}
+```
+
+Then, the CLI reads:
+
+```bash
+fetchd tx wasm instantiate <token_bin_code> '{"name": "yout_token_name", "symbol": "SYMBOL", "decimals": 6, ... }' --from your_key
+```
+
+After confirmation, your token is minted on the Terra network!
```
Comment (jhlee-young) [inline content/docs/howto/token.md]: Network name

File: content/docs/howto/query.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+- RESTFul API
+```URL
+<light_clinet_address>/cosmwasm/wasm/v1/contract/<contract_address>/smart/<JSON_formed_message_with_base64>
+```
+
+ex)
+
+```URL
+https://rest-dorado.fetch.ai/cosmwasm/wasm/v1/contract/terra1466nf3zuxpya8q9emxukd7vftaf6h4psr0a07srl5zw74zh84yjqxl5qul/smart/eyJwYWlycyI6e319
```
Comment (jhlee-young) [inline content/docs/howto/query.md]: contract address 👀

File: content/docs/howto/create_your_own_pair.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+## Instantiation by Contract Address
+
+You should use the Starfleit token factory contract.
+- fetchhub-4: `fetch1kmag3937lrl6dtsv29mlfsedzngl9egv5c3apnr468q50gu04zrqea398u`
```
Comment (jhlee-young) [inline content/docs/howto/create_your_own_pair.md]: a same address in dorado-1' one of contract_address above

File: content/docs/howto/swap.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+        }
+      },
+      "amount": "10"
+    },
+    "to": "<Addr>"
+  }
+}
+```
+
+`swap.offer_asset` represents your source asset. It is mandatory to acknowledge the decimal of the token setting when entering the value of `swap.offer_asset.amount` for an accurate transaction you want. For instance, the decimal of FET is 18, and it implies the value `10` of `swap.offer_asset.amount` expresses '10 x 10^-18` in the actual amount. That means you should multiply with the matching value, `10^(decimal)`. And the number can vary by other navie or IBC tokens but it cannot be detected if it is not acknowloged by human and the person doesn't register. Please find Starfleit team on [#starfleit channel on Fetch.ai Discord](https://bit.ly/3ra5uMI)
```
Comment (jhlee-young) [inline content/docs/howto/swap.md]: ' -> ` after expresses

Comment (jhlee-young) [reply content/docs/howto/swap.md]: typo `navie`

Comment (jhlee-young) [reply content/docs/howto/swap.md]: typo `acknowloged`

Comment (jhlee-young) [reply content/docs/howto/swap.md]: could you add a brief purpose of finding the team on Discord? it seems to be helpful to understand the context.

File: content/docs/howto/create_your_own_pair.md (hunk 1 above)
Comment (psy2848048) [reply content/docs/howto/create_your_own_pair.md]: Old news 😅  https://github.com/starfleit/docs-starfleit-io/pull/2/commits/7d89d5cad805b2631f8f3cf0eb202f8324f27815

File: content/docs/howto/token.md (hunk 2 above)
Comment (psy2848048) [reply content/docs/howto/token.md]: Old news 😅  https://github.com/starfleit/docs-starfleit-io/pull/2/commits/7d89d5cad805b2631f8f3cf0eb202f8324f27815

File: content/docs/howto/query.md (hunk 3 above)
Comment (psy2848048) [reply content/docs/howto/query.md]: Old news 😅  https://github.com/starfleit/docs-starfleit-io/pull/2/commits/7d89d5cad805b2631f8f3cf0eb202f8324f27815

File: content/docs/howto/swap.md (hunk 5 above)
Comment (psy2848048) [reply content/docs/howto/swap.md]: https://github.com/starfleit/docs-starfleit-io/pull/2/commits/f17150803f9fb91ffc9356a513627e638c543ba6

File: content/docs/howto/create_your_own_pair.md (hunk 4 above)
Comment (psy2848048) [reply content/docs/howto/create_your_own_pair.md]: https://github.com/starfleit/docs-starfleit-io/pull/2/commits/f17150803f9fb91ffc9356a513627e638c543ba6

**Review by jhlee-young — APPROVED** [review]

**Pull request merged**